Neo.Storage.RocksDB/Neo.Storage.RocksDB.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Neo" Version="3.0.0-CI00810" />
+    <PackageReference Include="RocksDbNative" Version="6.2.2" />
+    <PackageReference Include="RocksDbSharp" Version="6.2.2" />

```

Comment:
Can't we use our own releases?

Reply:
I think this package is officially provided by RocksDb.

https://github.com/facebook/rocksdb/blob/master/LANGUAGE-BINDINGS.md